hazelcast-integration/kubernetes/hazelcast-enterprise.yaml

```diff
@@ -3,21 +3,21 @@ kind: StatefulSet
 metadata:
   name: hazelcast
   labels:
-    app: hazelcast
+    app.kubernetes.io/name: hazelcast
```

Comment:
What is the main motivation of those type of changes?

Reply:
That's actually the new standard in Kubernetes, so it's just to keep the Kubernetes standards in our code samples.